src/main/java/com/assure/movie/model/domain/MovieCatalog.java

```diff
@@ -6,18 +6,16 @@
 
 @Entity
 @Table(
-    name = "movie_catalog",
-    uniqueConstraints= @UniqueConstraint(columnNames={"movie_id"})
+    name = "movie_catalog"
+    //uniqueConstraints= @UniqueConstraint(columnNames={"movie_id"})
 )
 public class MovieCatalog implements Identifiable<MovieCatalogId> {
+
     @EmbeddedId
     private MovieCatalogId id;
 
     @MapsId("movieId")
     @OneToOne(fetch = FetchType.LAZY)
-    @JoinColumn(
-            name = "movie_id",
-            referencedColumnName = "movie_id")
     private Movie movie;
 
     @OneToOne(fetch = FetchType.LAZY)
```

Comment:
Does this means that a Price only belongs to a single MovieCatalog?
Is not possible to have the same price for differents Movies in the Catalog?

Reply:
this part was a little confusing for me, because the price should be better as a column of the movie, but as the price now is a indepente table this table probably will have more information about price, probably discounts or other kind of information. 
I decided that this part is better as a relation one to one becuase If I want to change the price of a particular movie this will not affect the price of other movies.

Reply:
ok, thanks for the clarification.